tensorflow/lite/arena_planner_test.cc

```diff
@@ -210,6 +210,12 @@ TEST_F(ArenaPlannerTest, EmptyGraph) {
   Execute(0, 10);
 }
 
+TEST_F(ArenaPlannerTest, DeallocationOfTensor) {
```

Comment:
At least a comment here describing what we're testing, it's not at all obvious from the name of the test or the code here.

Reply:
@jdduke , sorry for the miss,i have updated the code with the comments, explaining why this TC is added and what problem previously was seen. Hope this is ok.

Regards
Amit